src/common/config_opts.h

```diff
@@ -1308,6 +1308,7 @@ OPTION(filestore_commit_timeout, OPT_FLOAT, 600)
 OPTION(filestore_fiemap_threshold, OPT_INT, 4096)
 OPTION(filestore_merge_threshold, OPT_INT, 10)
 OPTION(filestore_split_multiple, OPT_INT, 2)
+OPTION(filestore_split_rand_factor, OPT_U32, 0) // randomize the split threshold by adding 16 * [0, rand_factor)
```

Comment:
I suppose it can't hurt. We just don't have a great number to put here without good perf testing